source/type/pb_converter.cpp

```diff
@@ -429,6 +429,14 @@ void PbCreator::Filler::fill_pb_object(const ng::Admin* adm, pbnavitia::Administ
     if(!adm->insee.empty()){
         admin->set_insee(adm->insee);
     }
+    for (const auto& sa: adm->main_stop_areas) {
+        auto* pb_sa = admin->add_main_stop_areas();
+        fill_pb_object(sa, pb_sa);
+        for (const auto& sp: sa->stop_point_list) {
+            auto* pb_sp = pb_sa->add_stop_points();
+            fill_pb_object(sp, pb_sp);
+        }
+    }
```

Comment:
It can be great if we limit the size of that to the bare minimum. depth=0 for the 2 I think.